profile/README.md

```diff
@@ -154,6 +154,31 @@ jobs:
         uses: matlab-actions/run-tests@v2
 ```
 
+## Use Virtual Display on Linux Runner
+GitHub-hosted runners based on Linux&reg; do not provide a display. To run MATLAB code that requires a display, first set up a virtual display on the runner by starting an Xvfb display server. For example, set up a virtual server to run the tests for an app created in MATLAB.
```

Comment:
Can we add a "such as" after "To run MATLAB code that requires a display,"? Like:

To run MATLAB code that requires a display, such as UI app testing, ...

Reply:
Good idea. Updated as:

To run MATLAB code that requires a display, such as tests that verify the correctness of an app, ..